whirlpool/washerdryer.py

```diff
@@ -108,3 +108,6 @@ def get_cycle_status_rinsing(self):
 
     def get_cycle_status_spinning(self):
         return self.attr_value_to_bool(self.get_attribute(ATTR_CYCLE_STATUS_SPINNING))
+
+    def get_all_attributes(self):
+        return self._data_dict.get("attributes")
```

Comment:
Sorry, this isn't needed for the auth fix.  It's an artifact of some testing I was doing.